### PR TITLE
README.rst for app_opt/xgboost module [skip ci]

### PR DESCRIPTION
**Pull request by nvidianz**

### CODE REVIEW

**Comment by nvkevlu** [comment]
rst files should be in the docs structure in order to get built properly, so this will have to be moved. We still need to reorganize the docs anyway so we can put this anywhere now and move this later.

File: nvflare/app_opt/xgboost/README.rst [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+          "site-1b"
+        ],
+        "app_site2_gpu0": [
+          "site-2a"
+        ],
+        "app_site2_gpu1": [
+          "site-2b"
+        ]
+      },
+      "min_clients": 4
```
Comment (YuanTingHsieh) [inline nvflare/app_opt/xgboost/README.rst:218]: i need to check if this is "min_clients" or "mandatory_clients"

**Review by YuanTingHsieh — APPROVED** [review]
good explanation for the app_opt/xgboost code.
We can do unify documentation in later PRs.

File: nvflare/app_opt/xgboost/README.rst (hunk 1 above)
Comment (YuanTingHsieh) [reply nvflare/app_opt/xgboost/README.rst:218]: should be "mandatory_clients"

**Pull request merged**